src/app/pages/waiting-room-to-car/components/mot-components/mot-card/mot-card.component.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  Component, EventEmitter, Input, Output,
+} from '@angular/core';
+import { VehicleDetails } from '@providers/vehicle-details-api/vehicle-details-api.model';
+import { ConnectionStatus, NetworkStateProvider } from '@providers/network-state/network-state';
+import { UntypedFormGroup } from '@angular/forms';
+import { HttpStatusCodes } from '@shared/models/http-status-codes';
+import { MotStatusCodes } from '@shared/models/mot-status-codes';
+
+@Component({
+  selector: 'mot-card',
+  templateUrl: './mot-card.component.html',
+  styleUrls: ['./mot-card.component.scss'],
+})
+export class MotCardComponent {
+
+  @Input()
+  status: string = '';
+  @Input()
+  formGroup: UntypedFormGroup;
+  @Input()
+  data: VehicleDetails = {
+    registration: '',
+    make: '',
+    model: '',
+    colour: '',
+    status: '',
+    testExpiryDate: '',
+    testDueDate: '',
+    testDate: '',
+  };
+  alternateEvidenceRadioCheck: boolean;
+  @Output()
+  alternateEvidenceChange = new EventEmitter<boolean>();
+  @Output()
+  alternativeEvidenceDescriptionUpdate = new EventEmitter<string>();
+
+  constructor(
+    public networkState: NetworkStateProvider,
+  ) {
+  }
+
+  callWasSuccessful() {
+    return (+this.status === HttpStatusCodes.OK || this.status === 'Already Saved')
+      && this?.data?.status !== MotStatusCodes.NO_DETAILS
+      && this.networkState.getNetworkState() === ConnectionStatus.ONLINE;
+  }
+
+  noDetails(): boolean {
+    return (+this.status === HttpStatusCodes.NO_CONTENT
+      || this.data?.status === MotStatusCodes.NO_DETAILS);
+  }
+
+  is404(): boolean {
+    return (+this.status === HttpStatusCodes.NOT_FOUND);
+  }
+
+  isOffline(): boolean {
+    return (this.networkState.getNetworkState() !== ConnectionStatus.ONLINE);
+  }
+  isValidMOT() {
+    return this.data.status === MotStatusCodes.VALID;
+  }
```

Comment:
There is no reason we should be handling this inside child components, this should all be handled by the provider which gets the data and/or the basepage/parent.